docs/en/enterprise-edition/policy-reference/aws-policies/s3-policies/bc-aws-s3-24.adoc

```diff
@@ -17,63 +17,52 @@
 
 |Subtype
 |Build
-//, Run
 
 |Frameworks
 |Terraform,TerraformPlan
 
 |=== 
 
 
-
 === Description 
 
 
-If a bucket's versioning configuration is MFA Delete: enabled, the bucket owner must include the x-amz-mfa request header to delete an object.
-Requests that include x-amz-mfa must use HTTPS.
-Configuring a bucket to enable MFA (multi-factor authentication) Delete requires additional authentication for either of the following operations:  (1) Change the versioning state of your bucket  (2) Permanently delete an object version.
-We recommend you configure S3 bucket with MFA Delete: enabled.
-
-////
-=== Fix - Runtime
-
-
-* CLI Command* 
-
-
-
-
-[source,shell]
-----
-{
- "aws s3api put-bucket-versioning --profile my-root-profile --bucket my-bucket-name --versioning-configuration Status=Enabled,MFADelete=Enabled --mfa "arn:aws:iam::00000000:mfa/root-account-mfa-device 123456"",
+This policy is designed to prevent configuration policies that could potentially lock out non-root users from accessing S3 buckets. This check is vital for maintaining access control in AWS environments and ensuring that bucket policies do not inadvertently create security risks by overly restrictive configurations.
```

Comment:
From what you state, overly restrictive policies causing non-root users to be locked out from accessing S3 buckets would primarily pose an operational risk rather than a security risk.